include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -428,15 +428,16 @@ template <typename _KernelName, ::std::uint32_t __radix_bits, bool __is_ascendin
 sycl::event
 __radix_sort_reorder_submit(_ExecutionPolicy&& __exec, ::std::size_t __segments, ::std::size_t __block_size,
                             ::std::size_t __sg_size, ::std::uint32_t __radix_offset, _InRange&& __input_rng,
-                            _OutRange&& __output_rng, _OffsetBuf& __offset_buf, sycl::event __dependency_event
+                            _OutRange&& __output_rng, _OffsetBuf& __offset_buf, sycl::event __dependency_event,
+                            _Proj __proj
 #if _ONEDPL_COMPILE_KERNEL
                             ,
                             _Kernel& __kernel
 #endif
 )
 {
     // typedefs
-    using _InputT = oneapi::dpl::__internal::__value_t<_InRange>;
+    using _KeyT = oneapi::dpl::__internal::__key_t<_Proj, oneapi::dpl::__internal::__value_t<_InRange>>;
```

Comment:
`_KeyT` is unused. Probably keeping it as `_InputT` and do not do any changes on line 502 is a right thing to do